download.md

```diff
@@ -6,8 +6,53 @@ show_hero: true
 ---
 
 <div class="container">
+   <!-- Card deck -->
+    <div class="card-deck spacer-30">
+    <div class="card mb-4">
+        <div class="card-body text-center">
+            <h3 class="card-header-title mb-3">Trino on Kubernetes</h3>
+            <p class="card-text">Run Trino on 
+              <a href="https://kubernetes.io/docs/setup/">Kubernetes</a> using
+              the <a href="https://trinodb.github.io/charts/">Trino Helm chart</a>.
+              This allows you to deploy locally,
+              or running full-scale systems on the cloud.
+            </p>
+            <a class="btn btn-pink" href="docs/current/installation/kubernetes.html">
+                Try Trino on Kubernetes >>
+            </a>
+        </div>
+    </div>
+    <div class="card mb-4">
+        <div class="card-body text-center">
+            <h3 class="card-header-title mb-3">Run a Trino container</h3>
+            <p class="card-text">Start Trino using container tools like
+              <a href="https://docs.docker.com/get-started/overview/">Docker</a>.
+              Use this method to experiment with Trino without worrying about scalability
+              and orchestration.
+            </p>
+            <a class="btn btn-pink" href="docs/current/installation/containers.html">
+                Spin up Trino on Docker >>
+            </a>
+        </div>
+    </div>
+    <div class="card mb-4">
+        <div class="card-body text-center">
+            <h3 class="card-header-title mb-3">Deploy Trino manually</h3>
+            <p class="card-text">Use the tarball for fine-tuned deployments and
+              customizing the installment of Trino. Learn the fundamentals of
+              deploying a Trino cluster.
+            </p>
+            <a class="btn btn-pink" href="docs/current/installation/deployment.html">
+                Learn how to deploy Trino >>
+            </a>
+        </div>
+    </div>
+    <!-- Card -->
+    </div>
+    <!-- Card deck -->
   <div class="row spacer-60">
     <div class="col-md-12 text-center">
+       <h2>Downloads</h2>
```

Comment:
The spacing here is off .. the gap to the cards is too big compared to the gap above the title .. not sure how to adjust